articles/tech-resources/custom-bounce-domain.md

```diff
@@ -16,37 +16,36 @@ To configure a bounce domain, choose the subdomain you would like to use (e.g. m
 
 1. Register the domain with SparkPost by using the sending domains [create endpoint](https://developers.sparkpost.com/api/sending-domains.html#sending-domains-create-post)
-1. CNAME-verify the domain by using the sending domains [verify endpoint](https://developers.sparkpost.com/api/sending-domains.html#sending-domains-verify-post).  Here's an example API call to CNAME-verify the mail.example.com domain:
+2. CNAME-verify the domain by using the sending domains [verify endpoint](https://developers.sparkpost.com/api/sending-domains.html#sending-domains-verify-post).  Here's an example API call to CNAME-verify the mail.example.com domain:
 
     ```
-    POST /api/v1/sending-domains/mail.example.com/verify
+    POST /api/v1/sending-domains/mail.yourdomain.com/verify
 
     {
       "cname_verify" : true
     }
     ```
 
-1. Optionally set the domain as your account default bounce domain so that it is automatically used as the bounce domain for all messages sent through the account.  Here's an example API call to set mail.example.com as the account default bounce domain:
+3. Optionally set the domain as your account default bounce domain so that it is automatically used as the bounce domain for all messages sent through the account.  Here's an example API call to set mail.example.com as the account default bounce domain:
 
     ```
-    PUT /api/v1/sending-domains/mail.example.com
+    PUT /api/v1/sending-domains/mail.yourdomain.com
 
     {
       "is_default_bounce_domain" : true
     }
     ```
 
-Alternately, you may configure multiple CNAME-verified sending domains and select one as the bounce domain on a per message basis by including it in the [transmission `return_path`](https://developers.sparkpost.com/api/transmissions.html#header-transmission-attributes) field for transmission requests using the REST API, or by specifying it in the SMTP MAIL FROM command for SMTP injections.
+Alternatively, you may configure multiple CNAME-verified sending domains and select one as the bounce domain on a per message basis by including it in the [transmission `return_path`](https://developers.sparkpost.com/api/transmissions.html#header-transmission-attributes) field for transmission requests using the REST API, or by specifying it in the SMTP MAIL FROM command for SMTP injections.
 
 After CNAME-verifying or marking the domain as the account default bounce domain, there is up to a 5 minute delay [for compliance verification](https://www.sparkpost.com/docs/getting-started/requirements-for-sending-domains/) before the domain may be used as a message's bounce domain.
 
-If you prefer to use the SparkPost app, then you can create a single bounce domain that will be used for all of your outgoing mail.
+If you prefer to use the SparkPost UI instead of the API to verify a bounce domain, please use the following instructions. Please note, however, that you cannot specify a default bounce domain via the UI; that must be done via the sending-domains API endpoint.
```

Comment:
Can we hoist the UI instructions to the top? That seems like the more common path for people.

You can now set a default bounce domain the UI. There's a section named "Use as default" at the bottom of each sending domains page. ;)